src/AutoBalance.cpp

```diff
@@ -801,14 +853,72 @@ class AutoBalance_CommandScript : public CommandScript
             { "mapstat",          SEC_GAMEMASTER,                        true, &HandleABMapStatsCommand,                  "Shows current autobalance information for this map-" },
             { "creaturestat",     SEC_GAMEMASTER,                        true, &HandleABCreatureStatsCommand,             "Shows current autobalance information for selected creature." },
         };
+        if (allowPerGroupEnable)
+        {
+            ABCommandTable.push_back(ChatCommand("on",    SEC_PLAYER, false, &HandleABEnableCommand,  "Enable autobalance for your group."));
+            ABCommandTable.push_back(ChatCommand("off",   SEC_PLAYER, false, &HandleABDisableCommand, "Disable autobalance for your group."));
+            ABCommandTable.push_back(ChatCommand("state", SEC_PLAYER, false, &HandleABStateCommand,   "Show current autobalance state for your group."));
+        }
 
         static std::vector<ChatCommand> commandTable =
         {
-            { "autobalance",     SEC_GAMEMASTER,                            false, NULL,                      "", ABCommandTable },
+            { "autobalance",     SEC_PLAYER,                             true, NULL,                      "", ABCommandTable },
         };
         return commandTable;
     }
 
+    static bool HandleABStateCommand(ChatHandler* handler, const char* args)
+    {
+        Player *player = handler->GetSession()->GetPlayer();
+        if (NULL == player)
+            return false;
+        Player *groupLeader = GetGroupLeaderOrSelf(player);
+        // Should never happen, because when not in group, GetGroupOrSelf returns player
+        if (NULL == groupLeader)
+            return false;
+        AutoBalancePlayerInfo *playerABInfo = groupLeader->CustomData.GetDefault<AutoBalancePlayerInfo>("AutoBalancePlayerInfo");
+        // TODO: should use database to store localized strings
+        if (player->GetSession()->GetSessionDbcLocale() == LOCALE_frFR)
+            handler->PSendSysMessage("L'équilibrage automatique est %s.", playerABInfo->enabled ? "activé" : "désactivé");
+        else
+            handler->PSendSysMessage("Autobalance is %s.", playerABInfo->enabled ? "enabled" : "disabled");
+        return true;
+    }
+
+    static bool SetABAutobalance(ChatHandler *handler, bool enable)
```

Comment:
Maybe this should be named something like:

```suggestion
    static bool SetGroupAutobalance(ChatHandler *handler, bool enable)
```